src/include/network/peloton_rpc_handler_task.h

```diff
@@ -0,0 +1,52 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// peloton_rpc_handler_task.h
+//
+// Identification: src/include/network/peloton_rpc_handler_task.h
+//
+// Copyright (c) 2015-2018, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+#include "capnp/ez-rpc.h"
+#include "capnp/message.h"
+#include "common/dedicated_thread_task.h"
+#include "common/logger.h"
+#include "kj/debug.h"
+#include "peloton/capnp/peloton_service.capnp.h"
+
```

Comment:
Create a .cpp file and move the actual implementation of functions there as well.

Reply:
Holding this off until actual implementation of the index creation code.